hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetInfoHandler.java

```diff
@@ -165,8 +164,8 @@ private Optional<TokenInfo> infoForToken(
                 info.feeScheduleKey(token.feeScheduleKey());
             }
 
-            if (token.autoRenewAccountNumber() != 0) {
-                info.autoRenewAccount(AccountID.newBuilder().accountNum(token.autoRenewAccountNumber()));
+            if (token.autoRenewAccountId().accountNum() != 0) {
```

Comment:
probably should be checking `token.autoRenewAccountId() != null` here, rather than a 0 number.

Reply:
Agreed, now that we're using account IDs, we don't want to be using sentinel values of <= 0 anymore